src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
```

Comment:
Ya I don't love using `sleep` but I think that sometimes I'd have a problem without it.